docs/anonymizer/index.md

```diff
@@ -63,7 +63,7 @@ Persidio anonymizer comes with predefined anonymizers but can easily be extended
         text="My name is Bond, James Bond",
         analyzer_results=[RecognizerResult("PERSON", 11, 15, 0.8),
                           RecognizerResult("PERSON", 17, 27, 0.8)],
-        anonymizers_config={"PERSON": AnonymizerConfig("replace", {"new_value": "BIP"})}
+        operators={"PERSON": OperatorConfig("replace", {"new_value": "BIP"})}
```

Comment:
I think that the notebook has similar example that needs to be changed

Reply:
tnx.Changing.